### PR TITLE
added literal instead string concatenation

### LINKED ISSUE #124: (JS-0246) Require template literals instead of string concatenation
## Description
In ES2015 (ES6), we can use template literals instead of string concatenation.

## Occurrences
There are 2 occurrences of this issue in the repository.

See all occurrences on DeepSource &rarr; [deepsource.io/gh/lenstube-xyz/lenstube/issue/JS-0246/occurrences/](https://deepsource.io/gh/lenstube-xyz/lenstube/issue/JS-0246/occurrences/)

### PR DESCRIPTION
**Pull request by goverdhan07**
## Description
added literal instead string concatenation
Closes #124 

## Type of change

- [x] Minor changes in old functionality

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings

### CODE REVIEW

File: src/utils/functions/formatNumber.ts [hunk 1]
```diff
@@ -1,8 +1,9 @@
 export const formatNumber = (num: number) => {
-  if (num > 999 && num < 1000000) {
-    return (num / 1000).toFixed(0) + 'k'
+var str = 'k'
```
Comment (sasicodes) [inline src/utils/functions/formatNumber.ts:2]: we can use const variable

Comment (goverdhan07) [reply src/utils/functions/formatNumber.ts]: > we can use the const variable and a meaningful name 🙇🏻

Hi @sasicodes, updated the variable name to `attribute` , pls let me know if we can go ahead with this?

**Comment by sasicodes** [comment]
@goverdhan07 can you fix the comments ser? 🙇🏻

**Comment by zt4ff** [comment]
@sasicodes 
Can you assign this to me if it's still open?

**Comment by sasicodes** [comment]
Closing this because of inactivity. Feel free to reopen!

**Pull request closed without merging**